### ISSUE TITLE
ImageSwiper does not load any preview images

### ISSUE THREAD

**Issue by tufcos** [tagspaces/tagspaces#406]
I have downloaded the 2.1.1 version of tagspace but I can not view any images in ImageSwiper mode. The images load on all perspectives but I need the thumbnail versions. 
These errors have been submitted for android and ipad but I didn't see any issue for windows 7.

**Comment by XzK** [comment]
Hello @tufcos,

Here how to handle thumbnails: [TagSpaces 2.1](http://www.tagspaces.org/blog/tagspaces-2-1-released/#thumbnails)

**Comment by tufcos** [comment]
Thanks for the comment, I didn't know I had to first create the thumbnails myself. The link provides a thumbnail generation method for linux (also Mac codes are commented out). I'm using windows and I don't think that method will work for me. 
I've downloaded imagemagick but the code is not compatible. I don't know much coding, if anyone can help I'd appreciate it. My initial search for generating code with imagemagick in windows did not yield any good results. I'll post if I find any good thumbnail generating code within subfolders with png format.
